### PR TITLE
Izrada zadace

### PR DESCRIPTION
**Pull request by HarisJasarevic**

### CODE REVIEW

**Review by nejlamaksumic-abh — COMMENTED** [review]
Please pay attention to how to you name the Pull requests and working branches. Should be written in english and correspond with the task you're doing/problem you're solving. For example this PR could be named: `Automate PlaceLab smoke test`, and the branch name: `task3`.

File: src/test/java/com/PlaceLab/HarisJasarevic/qamp/tests/LoginTestWithEmptyCredentials.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package com.PlaceLab.HarisJasarevic.qamp.tests;
+
+import com.PlaceLab.HarisJasarevic.qamp.pages.LoginPage;
+import com.PlaceLab.HarisJasarevic.qamp.utils.WebDriverSetup;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+import org.testng.asserts.SoftAssert;
```
[the remark this answers is not in the document]
Comment (HarisJasarevic) [reply src/test/java/com/PlaceLab/HarisJasarevic/qamp/tests/LoginTestWithEmptyCredentials.java:9]: Will do.

File: src/test/java/com/PlaceLab/HarisJasarevic/qamp/tests/SinglePlaceSearchReport.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        driver = WebDriverSetup.getWebDriver(browser);
+        driver.get("https://demo.placelab.com/");
+        this.loginPage = new LoginPage(driver);
+        this.homePage = new HomePage(driver);
+        this.singlePlaceReportPage = new SinglePlaceReportPage(driver);
+        driver.manage().window().maximize();
+    }
+
+    @Parameters( { "email", "password" } )
+    @Test(priority = 6, description = "Validate user is able to login with valid credentials")
```
[the remark this answers is not in the document]
Comment (HarisJasarevic) [reply src/test/java/com/PlaceLab/HarisJasarevic/qamp/tests/SinglePlaceSearchReport.java:29]: Sorry, paste fail :)

**Comment by HarisJasarevic** [comment]
I'll explore that option, haven't done it before.

**Pull request merged**